src/components/item/MetadataField.vue

```diff
@@ -85,6 +88,10 @@
         type: String,
         default: ''
       },
+      metadataLanguage: {
```

Comment:
Given the component is named MetadataField, this prop could just be named "language".

Reply:
I think metadataLanguage is clearer since it's more specific and for trying to trace this back to it's origin. Since there are still other languages(UI, browser, fallback) that just 'language' could refer to even within this component and it's metadataLanguage elsewhere like in the record plugin and item template.